src/commands/app/deploy.js

```diff
@@ -111,9 +117,13 @@ class Deploy extends BuildCommand {
         if (v.app.hasFrontend && flags['web-assets']) {
           const opItems = getFilesCountWithExtension(v.web.distProd)
           const assetDeployedLogEvent = getAuditLogEvent(flags, aioConfig.project, 'AB_APP_ASSETS_DEPLOYED')
-          if (assetDeployedLogEvent) {
+          if (assetDeployedLogEvent && cliDetails) {
             assetDeployedLogEvent.data.opItems = opItems
-            await sendAuditLogs(cliDetails.accessToken, assetDeployedLogEvent, cliDetails.env)
+            try {
+              await sendAuditLogs(cliDetails.accessToken, logEvent, cliDetails.env)
+            } catch (error) {
+              this.log(chalk.red(chalk.bold('Error: Audit Log Service Error: Failed to send audit log event for deployment.')))
```

Comment:
it is an error in sending, but to the cli is it an error? I would say it's more like a warning here. Just do a `this.warn` without all the `chalk` adornment